### ISSUE TITLE
can't list actors or get client from gedis server

### ISSUE THREAD

**Issue by Dinaamagdy** [threefoldtech/jumpscaleX_core#128]
## Steps
gedis = j.servers.gedis.get('test', port=9901)
gedis.actor_add("/sandbox/code/github/threefoldtech/jumpscaleX_core/JumpscaleCore/servers/gedis/pytests/test_package/actors/simple.py", namespace='demo')
gedis.save()
- in tmux session using same gedis instance"j.servers.gedis.get('test').start()"

## Issue 
gedis.actors_list() return empty list so gedis.get_client doen't work

**Comment by serboctor** [comment]
@Dinaamagdy `actors_list` takes the parameter `namespace`, the default value is `default`. In your case, you need to pass `demo` to return actors in this namespace, or `None` so it returns actors of all namespaces.

**Comment by serboctor** [comment]
@Dinaamagdy Also, when you open the tmux to start the gedis server, you need to add the actors first. Actors are not part of the schema, and they aren't saved in the server instance, so you need to add them in the same shell right before starting the server.
`client_get` also takes `namespace` argument, the default is `default`, so you need to specify `demo` in your case.
I tested the whole scenario and everything works fine.

**Comment by Dinaamagdy** [comment]
I tried this scenario 
-  server=j.servers.gedis.get(name="test",port=123)  
 -server.actor_add("/sandbox/code/github/threefoldtech/jumpscaleX_core/JumpscaleCore/servers/gedis/pytests/test_package/actors/simple.py", namespace='demo')  
- server.save()
- server.start()
##### all in same shell then in another shell get the same instance of this server
server=j.servers.gedis.get(name="test")
server.actors  : don'y have the actors
server.actors_list("demo")  :return empty list

**Comment by serboctor** [comment]
the server instance doesnt save the list of actors, this is saved in memory. So once you open another shell, and get the server instance again, you wont really have access to the actors. What you should do instead, is get the gedis client, and use the actor directly.

In the second shell, you can do the following:
```
JSX> server=j.servers.gedis.get(name="test",port=123)                                                                                                                                                       
JSX> client = server.client_get("demo")                                                                                                                                                                     
JSX> client.actors                                                                                                                                                                                          
## gedisclientactors GedisClientActors






JSX> client.actors.simple.ping()                                                                                                                                                                            
b'PONG'
```
client.actors doesn't reallly print the actors, you can just execute the actor like I do.

I pushed a small fix to the client_get https://github.com/threefoldtech/jumpscaleX_core/commit/65c230e443df89be8f3c97ad8fb6f536dc52766a

**Comment by siddiquagig** [comment]
Verified

In the first shell

![image](https://user-images.githubusercontent.com/25789724/69028601-f1634200-09eb-11ea-8f62-6877bf73273f.png)


In the second shell, did this :

![image](https://user-images.githubusercontent.com/25789724/69028974-1d32f780-09ed-11ea-9689-1a4a19c6e1a1.png)
